docs/reference/function/add_logger.txt

```diff
@@ -35,7 +35,7 @@ Behavior
 --------
 
 This function allows the application to register one or more
-`Psr\\Log\\LoggerInterface <https://www.php-fig.org/psr/psr-3/#3-psrlogloggerinterface>`__
+`Psr\Log\LoggerInterface <https://www.php-fig.org/psr/psr-3/#3-psrlogloggerinterface>`__
```

Comment:
@norareidy: I assume the escaping here also needs to state since this is a basic link? Similar to what you had to do in https://github.com/mongodb/mongo-php-library/pull/1244/files#diff-0494d8195d629119e6de652538a76bd23063d2234cf08cad50eeebc94e6dee99R34